SourceCode/raytracer/raytracer/Animation.h

```diff
@@ -0,0 +1,109 @@
+#pragma once
+#include "CRTTypes.h"
+#include <memory>
+
+struct SceneObject;
+
+class Animation
```

Comment:
Documentation comment would be nice, especially for classes with a generic name :)